source/features/add-toggle-files-button.js

```diff
@@ -0,0 +1,18 @@
+import {h} from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate';
+import * as icons from '../libs/icons';
+
+export default function () {
+	select('.commit-tease .float-right').appendChild(<button
+		class="btn-octicon p-1 pr-2 js-toggle-files"
+		aria-label="Toggle files section"
+		aria-expanded="true">
+		{icons.arrowUp()}
+		{icons.arrowDown()}
```

Comment:
Instead of two separate SVGs, perhaps we can handle this with `transform: rotate(180deg)`